docs/tutorials/HD143006_Tut_P1.py

```diff
@@ -0,0 +1,164 @@
+# 
+# *Note: when `robust=-2.0` the result is similar to that of the Uniform scale*
+
+img, beam = gridder.get_dirty_image(weighting='uniform')
+img1, beam1 = gridder.get_dirty_image(weighting="briggs", robust=1.0, unit="Jy/arcsec^2")
+img2, beam2 = gridder.get_dirty_image(weighting="briggs", robust=0.0, unit="Jy/arcsec^2")
+img3, beam3 = gridder.get_dirty_image(weighting="briggs", robust=-1.0, unit="Jy/arcsec^2")
+
+# Great! Now let's make a plotting function to show us the MPoL image. If you have read through other MPoL tutorials, then this code should look familiar. We are going to plot all four of the different weightings, so creating a plotting function simplifies our code a lot.
+
+def plot(img, savename="image.png", imtitle="image"):
+    kw = {"origin": "lower", "extent": gridder.coords.img_ext}
+    fig, ax = plt.subplots(ncols=1)
+    im = ax.imshow(np.squeeze(img), **kw)
+    plt.colorbar(im)
+    ax.set_title(imtitle)
+    ax.set_xlabel(r"$\Delta \alpha \cos \delta$ [${}^{\prime\prime}$]")
+    ax.set_ylabel(r"$\Delta \delta$ [${}^{\prime\prime}$]")
+    fig.savefig(savename, dpi=300)
+    plt.xlim(left=-.75, right=.75)
+    plt.ylim(bottom=-.75, top=.75)
+    return ax
+
+plot(img, savename="uniform.png", imtitle="uniform")
+plot(img1, savename="robust_1.0.png", imtitle="robust_1.0")
+plot(img2, savename="robust_0.png", imtitle="robust_0")
+plot(img3, savename="robust_-1.0.png", imtitle="robust_-1.0")
+
+# Below we plot the DSHARP CLEAN image alongside the MPoL RML Dirty Image weighted with the Briggs scale and `robust=0.0` for comparison.
+
+kw = {"origin": "lower", "extent": gridder.coords.img_ext}
+fig, ax = plt.subplots(ncols = 2)
+ax[0].imshow(np.squeeze(clean_fits), **kw)
+ax[0].set_title('DSHARP CLEAN Image')
+ax[0].set_xlim(left=-.75, right=.75)
+ax[0].set_ylim(bottom=-.75, top=.75)
+ax[0].set_xlabel(r"$\Delta \alpha \cos \delta$ [${}^{\prime\prime}$]")
+ax[0].set_ylabel(r"$\Delta \delta$ [${}^{\prime\prime}$]")
+ax[1].imshow(np.squeeze(img2), **kw)
+ax[1].set_title('MPoL RML Dirty Image')
+ax[1].set_xlim(left=-.75, right=.75)
+ax[1].set_ylim(bottom=-.75, top=.75)
+ax[1].set_xlabel(r"$\Delta \alpha \cos \delta$ [${}^{\prime\prime}$]")
+ax[1].set_ylabel(r"$\Delta \delta$ [${}^{\prime\prime}$]")
+plt.tight_layout()
```

Comment:
It would be helpful to play with the plotting to see if we can get these figures to show up larger, perhaps as a two-row figure? The specific format of the documentation site has a pretty narrow horizontal space.